src/package-index/src/error.rs

```diff
@@ -22,7 +22,7 @@ pub enum Error {
     #[error("Failed to parse URL")]
     UrlParseError(#[from] url::ParseError),
     #[error("Invalid platform {0}")]
-    InvalidPlatform(String),
+    InvalidTarget(String),
```

Comment:
will this requires bump up to `0.2.0`?

Reply:
Hmm, possibly, yeah. I can revert that if we don't want to deal with that right now

Reply:
Preferable to minimize version changes unless it is necessary